app/models/data.py

```diff
@@ -242,10 +242,38 @@ def totalTestResults(self):
     def parse_str_to_date(date_input):
         return parser.parse(str(date_input), ignoretz=True).date()
 
-    def __init__(self, **kwargs):
-        # strip any empty string fields from kwargs
-        kwargs = {k: v for k, v in kwargs.items() if v is not None and v != ""}
+    def field_diffs(self, dict_other):
+        ''' Return the list of fields that dict_other would modify if applied
+        on this row.
+        Some business logic is applied, and some fields are skipped from comparison, field
+        aliases get special treatment.
+
+        Return: List[str], empty list if the dict results in no changes to current data
+        '''
+        fields = []
+        if not dict_other:
+            return fields
+
+        # we want to compare after all parsing is done
+        other = CoreData(**dict_other)
+
+        # special casing for aliasse (maybe move elsewhere?)
+        if 'lastUpdateIsoUtc' in dict_other:
```

Comment:
what if `lastUpdateTime` is also in `dict_other`? Probably worth only overriding if it doesn't contain `lastUpdateTime`

Reply:
that makes sense. In that case may as well combine into a single if condition, `if 'lastUpdateIsoUtc' in dict_other and not 'lastUpdateTime' in dict_other`